### PR TITLE
Roundstart AT Cleanup mk2

### PR DESCRIPTION
**Pull request by ShizCalev**

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -947,9 +947,7 @@
 /area/security/prison)
 "acf" = (
 /obj/effect/turf_decal/stripes/line,
-/turf/open/floor/plating{
-	icon_plating = "warnplate"
```
Comment (Shadowlight213) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Is removing this going to cause issues? Or is it an invalid sprite?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -81218,10 +81194,7 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 4
 	},
-/turf/open/floor/plating{
-	luminosity = 2;
```
Comment (Shadowlight213) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Did you miss keeping luminosity 2 here?

**Review by Shadowlight213 — COMMENTED** [review]
Found a couple of things that might be unintended changes.
Emphasis on might. Can't really tell without finding/looking at those tiles in dm.

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 2 above)
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Was a misplaced turf in the aux base pod room.

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: All those tiles have proper decals on them now. Pretty sure they were just leftover.

**Pull request merged**